webui/react/src/pages/ExperimentDetails/ExperimentActions.tsx

```diff
@@ -64,12 +64,12 @@ const ExperimentActions: React.FC<Props> = ({ experiment, onClick, onSettled }:
   }, [ experiment.id, onSettled ]);
 
   const handleUnarchive = useCallback(() => async (): Promise<void> => {
-    setBtnLoadingStates(state => ({ ...state, Archive: false }));
+    setBtnLoadingStates(state => ({ ...state, Archive: true }));
```

Comment:
**suggestion:** I think we may have missed having `Unarchive` as part of `Action` enum. If we have that, we can then use that instead of reusing `Archive` for the `handleUnarchive` handler.

Reply:
Updated :)